configuration/concepts/lab_tests.csv

```diff
@@ -95,27 +100,76 @@ d3966267-5882-437b-ae14-18dee9657d14,,,RBCs,Hematíes,Hematíes,RBCs,,LabTest,Co
+246c04c0-08a0-405b-890b-bdaa1d361cd8,,,Protozoa,protozoos,protozoos,Protozoa,,LabSet,N/A,,,Entamoeba coli;Entamoeba hartmanni;Iodamoeba butschhlii;Blastocystis hominis;Entamoeba histolytica;Balantidium coli;Giardia lamblia;Chilomastix mesnili;Trichomonas hominis,,,,,,,
+8d5d72c3-0abc-4bd3-a76a-2ee0b74e13b4,,,Ascaris lumbricoides,Ascaris lumbricoides,Ascaris lumbricoides,Ascaris lumbricoides,,LabTest,Text,,,,,,,,,,
+9933fa8d-0d9d-4a87-bb16-29c6748fd0a4,,,Trichuris trichiura,Trichuris trichiura,Trichuris trichiura,Trichuris trichiura,,LabTest,Text,,,,,,,,,,
+4cb726b0-d6a9-48e5-a8d9-d693ef6840d0,,,Hymenolepis nana,Hymenolepis nana,Hymenolepis nana,Hymenolepis nana,,LabTest,Text,,,,,,,,,,
+2247bba4-a433-44c3-a091-ea026fdf8e06,,,Taenia sp,Taenia sp,Taenia sp,Taenia sp,,LabTest,Text,,,,,,,,,,
+0733fbb6-41fa-457b-a57f-d6653caa85f8,,,Helminths,Helmintos,Helmintos,Helminths,,LabSet,N/A,,,Ascaris lumbricoides;Trichuris trichiura;Hymenolepis nana;Taenia sp,,,,,,,
+87b1df6c-2cb3-4e76-bb6a-d89045f35f07,,,Wright Test,Prueba de Wright,Prueba de Wright,Wright Test,,LabTest,Text,,,,,,,,,,
+64bddc56-09c6-4eb7-8e31-baabc15c0f50,,,Crystals,Cristales,Cristales,Crystals,,LabTest,Text,,,,,,,,,,
+d8d5293e-9289-484a-b969-05659c076e38,,,Casts,moldes,moldes,Casts,,LabTest,Text,,,,,,,,,,
+f72aca5b-7f99-44b7-b2ce-cba87411bafa,,,Aspect,Aspecto,Aspecto,Aspect,,LabTest,Text,,,,,,,,,,
+ffb5daf7-b0c2-4fd7-afad-9173ce5d3b75,,,Color,Color,Color,Color,,LabTest,Text,,,,,,,,,,
+8a61c5ee-a83c-4b48-a30b-a432c6e2409e,,,Yeast Cells,Células de levadura,Células de levadura,Yeast Cells,,LabTest,Coded,,Negative; Trace; Small; Moderate; Large,,,,,,,,
+11eb2dd2-fa80-4560-bd5d-bffe2d0ce064,,,Mucus,Moco,Moco,Mucus,,LabTest,Coded,,Negative; Trace; Small; Moderate; Large,,,,,,,,
+cf1cff9a-0df8-4e2d-8abf-f96a3392f8f0,,,Red Blood Cells in Urine,Glóbulos rojos en la orina,Glóbulos rojos en la orina,Red Blood Cells in Urine,,LabTest,Numeric,,,,,cells per field,,,,,
+ef036fb3-6e8c-4c79-9055-54913e4ef70f,,,White Blood Cells in Urine,Glóbulos blancos en la orina,Glóbulos blancos en la orina,White Blood Cells in Urine,,LabTest,Numeric,,,,,cells per field,,,,,
+9a89b6e4-143b-4c03-b929-b085621f75e5,,,Epithelial Tissue,Tejido epitelial,Tejido epitelial,Epithelial Tissue,,LabTest,Coded,,Negative; Trace; Small; Moderate; Large,,,,,,,,
+3fed1729-e4e3-434c-946c-58b3f9e8d963,,,Bacterial Cells,Células bacterianas,Células bacterianas,Bacterial Cells,,LabTest,Coded,,Negative; Trace; Small; Moderate; Large,,,,,,,,
+57bf229a-b469-49f1-9d29-3f6e90120b8d,,,Other Test Observations,Otras observaciones de prueba,Otras observaciones de prueba,Other Test Observations,,LabTest,Text,,,,,,,,,,
+666bee99-7881-418d-8ea1-5d2301aa0640,,,Microscopy Set,Juego de microscopía,Juego de microscopía,Microscopy Set,,LabSet,N/A,,,Crystals;Casts;Aspect;Color;Yeast Cells;Bacteria;Mucus;Red Blood Cells in Urine;White Blood Cells in Urine;Epithelial Tissue;Other Test Observations,,,,,,,

```

Comment:
Remove **aspect** and **colour** from this set and add it to **Urinalysis** and endeavours to have them as the first in the list

Reply:
Done